scripts/khiops-package-version

```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env bash
+# Creates a "package-version" of Khiops
+# - if the commit is tagged:
+#   - if the tag matches the Khiops version in the sources:
+#     - package-version = version
+#   - otherwise:
+#     - warn
+#     - package-version = version-SNAPSHOT-$TAG
+# - otherwise:
+#   - package-version = version-SNAPSHOT-$SHORT_COMMIT_HASH
+
+# Common safeguards
+set -euo pipefail
+
+# Save this script's directory
+SCRIPT_DIR="$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd)"
+
+main () {
+  # The ref_name it should be either a hash or a tag
+  declare ref_name="$1"
+
+  # Whether ref_name is a tag (default false)
+  declare is_tag="${2:-false}"
+
+  # Obtain the khiops version from the source
+  local script_dir
+  local khiops_version
+  khiops_version="$("$SCRIPT_DIR/khiops-version")"
+
+  # See the header of the script for this part
+  local khiops_package_version
+  if [[ "$is_tag" == "true" ]]
+  then
+    if [[ "$ref_name" == "v${khiops_version}" ]]
+    then
+      khiops_package_version="$khiops_version"
+    else
+      khiops_package_version="$khiops_version-SNAPSHOT-$ref_name"
+      echo "::warning: Tag '$ref_name' doesn't match the khiops source version '$khiops_version'" 1>&2
+      echo "::warning: Creating snapshot package $khiops_package_version" 1>&2
```

Comment:
on ne crée pas de package ici, si ?

Reply:
You are right. How about: `Creating preview version "$khiops_package_version"`

Reply:
Done